pgd_splicer/ProcessPDBTask.py

```diff
@@ -405,8 +405,48 @@ def parseWithBioPython(path, props, chains_filter=None):
     # Be kind; rewind.
     decompressed.seek(0)
 
+    alt_structure = Bio.PDB.PDBParser().get_structure('pdbname',
+                                                  decompressed.name)
+    io=PDBIO()
+    io.set_structure(alt_structure)
+    io.save("not-ordered.pdb")
+
+
+    for ichain in alt_structure[0] :
+        for ires in ichain :
+            if ires.is_disordered():
+                res_sum = {}
+                count_sum = {}
+                for atom in ires :
+                    if atom.is_disordered():
+                        try :
+                            res_sum[ atom.get_altloc() ] = res_sum[atom.get_altloc()] + atom.get_occupancy()
```

Comment:
Also, a smart person recently suggested to me that you consider a single data structure instead of two hashes.

What do you think of having a dict named altloc, with keys being atom.get_altloc() output and values being {'sum': 0, 'count': 0} ?  This would make the KeyError stuff cleaner.

One more thought:  call atom.get_altloc() once and save the value?